common/views/themes/default.js

```diff
@@ -10,6 +10,12 @@ const theme = {
     large: 960,
     xlarge: 1338,
   },
+  containerPadding: {
+    small: 18,
+    medium: 42,
+    large: 60,
+    xlarge: 60,
```

Comment:
Could you run me through what these are and when to use them?

Reply:
Have removed this and used a container div to wrap the preview instead.